### ISSUE TITLE
White flash while tab browsing

### ISSUE THREAD

**Issue by Davincible** [deanoemcke/thegreatsuspender#1034]
Please complete the following information when submitting a feature request or bug report.
 * Extension version: v7.1.0
 * Browser name & version: browsing
 * Operating system & version: Pop OS 19.04 - kernel 5.3

I love darkmode, however when I'm browsing tabs with alt+tab, sometimes the pages flash white for a milisec, and go back to darkmode again. I also see this behaviour with a light theme, however then it's not as visible. The flashes are quite annoying.

**Comment by roko-p** [comment]
I was just going to create an issue for this.
Same thing on MacOS (Mojave 10.14.6), same extension version (7.1.0) in Chrome (77.0.3865.90 (Official Build) (64-bit))

**Comment by ryanlynx** [comment]
This is happening also on Win 10 Pro(1903), same extension version (7.1.0) in Brave (Version 0.69.135 Chromium: 77.0.3865.120, Official Build, 64-bit), using browser flag 'Dark mode'. 
At night time the white flashing is extra irritating and eye straining.

**Comment by ryanlynx** [comment]
> This is happening also on Win 10 Pro(1903), same extension version (7.1.0) in Brave (Version 0.69.135 Chromium: 77.0.3865.120, Official Build, 64-bit), using browser flag 'Dark mode'.
> At night time the white flashing is extra irritating and eye straining.

So one interim fix I found was to not use 'dark mode' that is found under the settings for this plugin.

**Comment by wisteso** [comment]
Yeah this white flash is quite jarring, especially for those of us that work at night.

**Comment by Davincible** [comment]
Exactly... still no response though. Think I'll look for a tab suspender without this issue

**Comment by MichelDiz** [comment]
Just enable this and any white flash is gone. 
![Captura de Tela 2019-11-10 às 18 07 58](https://user-images.githubusercontent.com/6570955/68550800-156bc580-03e5-11ea-9a89-cca4007b71d6.png)

Cheers.

**Comment by wisteso** [comment]
Why is this unconfirmed? Multiple users are reporting this. Though seems we have a workaround with the chrome flags

**Comment by davidneedham** [comment]
I'm still seeing this bug on osx 10.15.7 running Google Chrome Version 87.0.4280.88 (Official Build) (x86_64) with greatsuspender version v7.1.8.

I tried the temporary workaround to tell chrome to force dark mode, but it's no good. It does weird things on YouTube and there's still a flash when switching between two suspended tabs. It just flashes to black instead of flashing to white.
